R/autoplot.R

```diff
@@ -25,19 +25,21 @@
 #' @return A ggplot object
 #' @export
 #'
+#' @importFrom checkmate assert assert_int anyInfinite assert_character
```

Comment:
One minor suggestion: rather than putting these imports with the function they're used in, maybe better to put them all in a common location? That way we don't end up importing the same one in the Roxygen 15 different times.